gensim/corpora/dictionary.py

```diff
@@ -147,20 +153,23 @@ def doc2bow(self, document, allow_update=False, return_missing=False):
 
         token2id = self.token2id
         if allow_update or return_missing:
-            missing = dict((w, freq) for w, freq in iteritems(counter) if w not in token2id)
+            missing = dict((w, freq)
```

Comment:
No vertical indent in gensim. Use hanging indent.

This is the last comment but many instances of the same issues (unwanted line breaks, vertical indent) elsewhere in the proposed changes.